### PR TITLE
docs: Adding the missed documentation for the RBAC

### PR DESCRIPTION
**Pull request by lokeshrangineni**
Adding the missed documentation for - 
* OIDC Token requirement or assumptions
* Added `feast permissions check` cli command documentation.

### CODE REVIEW

**Review by redhatHameed — APPROVED** [review]

File: docs/getting-started/components/authz_manager.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Authorization Manager
-An Authorization Manager is an instance of the `AuthManager` class that is plugged into one of the Feast servers to extract user details from the current request and inject them into the [permissions](../../getting-started/concepts/permissions.md) framework.
+An Authorization Manager is an instance of the `AuthManager` class that is plugged into one of the Feast servers to extract user details from the current request and inject them into the permission framework.
```
Comment (franciscojavierarceo) [inline docs/getting-started/components/authz_manager.md:2]: why remove link?

Comment (lokeshrangineni) [reply docs/getting-started/components/authz_manager.md]: original link is broken (extra s in the file permissions.md). Thought we moved that doc is moved but with your comment realized it is type or some refactoring. now fixed it with the correct link.

File: docs/getting-started/components/authz_manager.md [hunk 2]
```diff
@@ -44,7 +44,10 @@ The server, in turn, uses the same OIDC server to validate the token and extract
 
 Some assumptions are made in the OIDC server configuration:
 * The OIDC token refers to a client with roles matching the RBAC roles of the configured `Permission`s (*)
-* The roles are exposed in the access token passed to the server
+* The roles are exposed in the access token passed to the server
```
Comment (franciscojavierarceo) [inline docs/getting-started/components/authz_manager.md:47]:
```suggestion
* The roles are exposed in the access token that is passed to the server 
```

File: docs/getting-started/components/authz_manager.md [hunk 3]
```diff
@@ -44,7 +44,10 @@ The server, in turn, uses the same OIDC server to validate the token and extract
 
 Some assumptions are made in the OIDC server configuration:
 * The OIDC token refers to a client with roles matching the RBAC roles of the configured `Permission`s (*)
-* The roles are exposed in the access token passed to the server
+* The roles are exposed in the access token passed to the server 
+* JWT token is expected to have verified signature and not expired. Feast oidc token parser logic validates for `verify_signature` and `verify_exp` so make sure that given OIDC provider is configured to meet this requirement.
```
Comment (franciscojavierarceo) [inline docs/getting-started/components/authz_manager.md:48]:
```suggestion
* The JWT token is expected to have a verified signature and not be expired. The Feast OIDC token parser logic validates for `verify_signature` and `verify_exp` so make sure that the given OIDC provider is configured to meet these requirements.
```

File: docs/getting-started/components/authz_manager.md [hunk 4]
```diff
@@ -44,7 +44,10 @@ The server, in turn, uses the same OIDC server to validate the token and extract
...
+* preferred_username should be part of the jwt token claim.
```
Comment (franciscojavierarceo) [inline docs/getting-started/components/authz_manager.md:49]:
```suggestion
* The preferred_username should be part of the JWT token claim.
```

**Review by franciscojavierarceo — APPROVED** [review]
left some small nits, but this looks great! Thank you!

**Pull request merged**